### PR TITLE
flight: osd: various small cleanups

### PR DESCRIPTION
**Pull request by mlyle**
[Review on Reviewable](https://reviewable.io/reviews/d-ronin/dronin/667)

### CODE REVIEW

File: flight/Modules/OnScreenDisplay/onscreendisplay.c [hunk 1]
```diff
@@ -1276,6 +1263,8 @@ void render_user_page(OnScreenDisplayPageSettingsData * page)
 		AccelsData accelsData;
 		AccelsGet(&accelsData);
 		// apply low pass filter to reduce noise bias
+		static AccelsData accelsDataAcc = {};
```
Comment (tracernz) [inline flight/Modules/OnScreenDisplay/onscreendisplay.c]: Empty initializer lists are naughty right? http://stackoverflow.com/questions/17589533/is-an-empty-initializer-list-valid-c-code

Comment (mlyle) [inline flight/Modules/OnScreenDisplay/onscreendisplay.c]: It indeed is a gcc-ism.  But imo preferable to coercing a 0 in there or increasing scope and using bzero.

Comment (mlyle) [inline flight/Modules/OnScreenDisplay/onscreendisplay.c]: aight I relented and put a 0 in :P

**Comment by mluessi** [comment]
Code looks good to me. Let me know when I should test this.

**Comment by mlyle** [comment]
I think it's ready-- whenever you like (or I can when I'm back).

**Comment by mluessi** [comment]
Bench tested, works as expected. Thanks @mlyle

**Pull request merged**